assets/RandomComicViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RandomComicViewController.swift
+//  xkcd
+//
+//  Created by Ekam Singh Dhaliwal on 2019-09-04.
+//
+
+import UIKit
+
+class RandomComicViewController: UIViewController {
+
+  @IBOutlet weak var comicImage: UIImageView!
+
+
+  override func viewDidLoad() {
+        super.viewDidLoad()
+
+
+    }
```

Comment:
Is swiftLint enabled/running properly? I would think it would through a warning about too much white space here.